### PR TITLE
send_receipt: handle email backend returning None

### PR DESCRIPTION
**Pull request by blueyed**
TODO:

- [ ] test

### CODE REVIEW

File: pinax/stripe/hooks.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def send_receipt(self, charge, email=None):
                 to=[email],
                 from_email=settings.PINAX_STRIPE_INVOICE_FROM_EMAIL
             ).send()
-            charge.receipt_sent = num_sent > 0
+            charge.receipt_sent = num_sent and num_sent > 0
```
Comment (ticosax) [inline pinax/stripe/hooks.py]: You could do that also `bool(num_sent) > 0`.
reads better I think.

Comment (blueyed) [reply pinax/stripe/hooks.py]: Not motivated enough to change that now.

**Comment by blueyed** [comment]
ready from my PIV: there could be a test, but not from me.

File: pinax/stripe/hooks.py (hunk 1 above)
Comment (ticosax) [reply pinax/stripe/hooks.py]: fair enough

**Pull request merged**